### PR TITLE
New Menu and Fix CLI+ [Authentication]

### PR DESCRIPTION
**Pull request by HaveFUNRICH**
# CLI+
## Changelog
[Critical](https://www.youtube.com/watch?v=dQw4w9WgXcQ)
#### Readme.md
 * Rename Salad.Authentication to sAccessToken
 * Add sIdRefreshToken
 * Remove SaladCLI+ is broken
#### Authentication.py
 * Rename Salad.Authenctication to sAccessToken
 * Add sIdRefreshToken Cookie
#### Read me!.txt
 * Fix grammar (*depencies* to dependencies)
 * Add sIdRefreshToken

## Draft checking
[Low](https://www.youtube.com/watch?v=dQw4w9WgXcQ)
#### Authentication.py
 * Rename sldRefresh to sIdRefreshToken (line 24)
 * Rename RefreshToken to sldRefreshToken (line 23)

## Open
[None](https://www.youtube.com/watch?v=dQw4w9WgXcQ)
#### Removed the read me!.txt upgrade
 * Command by Mr. Furry 3oF
#### Anthentication.py
 * Vukky Bukky Compatible only for auth.py
 * Add sIdRefreshToken to line 46 and 47
 * Revert to original 
#### Start.py
 * Get session
#### Menu.py
 * Show how much do you need to level up
 * Show current level
 * Show the current session
#### Readme.md
 * Remove the bad sector of @HaveFUNRICH Brain

### CODE REVIEW

**Review by Tpelaaja — APPROVED** [review]

**Review by Tpelaaja — APPROVED** [review]
walker kneecaps are going to be shit

**Comment by HaveFUNRICH** [comment]
breh it change like sIdRefresh and sAccessToken every 5 min

**Comment by HaveFUNRICH** [comment]
> breh it change like sIdRefresh and sAccessToken every 5 min

I totally did not enter salad on my second accounts so it just did this

**Comment by HaveFUNRICH** [comment]
@Vukkyy test
https://www.youtube.com/watch?v=dQw4w9WgXcQ

File: Salad CLI+/Read Me!.txt [hunk 1]
```diff
@@ -61,8 +61,8 @@ Login with your Salad account.
 Click on the Cookies icon (Should look like an lock on firefox) on the left side of the Search bar.
 Click on "Cookies".
 Open the app-api.salad.io folder in the cookies.
-Open Salad.Authentication.
-Right-Click on the Content in the Salad.Authentication cookie, and click Select-all.
+Open sAccessToken.
+Right-Click on the Content in the sAccessToken cookie, and click Select-all.
```
Comment (Vukkyy) [inline Salad CLI+/Read Me!.txt:65]: Don't you need both?

**Review by Vukkyy — CHANGES_REQUESTED** [review]

File: Salad CLI+/options/authentication.py [hunk 2]
```diff
@@ -10,17 +10,18 @@
     import requests
     import dateutil.parser
 except ImportError:
-    print("One or more Modules not found. Press enter to install! After installing please restart Salad CLI+")
-    input()
+    print("Installing important software, please wait...")
     os.system("pip install -r ./Options/requirements.txt")
+    os.system("python Start.py")
```
Comment (Vukkyy) [inline Salad CLI+/options/authentication.py:15]: This is my change, not yours, as indicated in your PR description. It's already in #46 and merging that will work perfectly fine - please remove it from here. Thank yous!

**Review by Vukkyy — CHANGES_REQUESTED** [review]

File: Salad CLI+/options/authentication.py (hunk 2 above)
Comment (HaveFUNRICH) [reply Salad CLI+/options/authentication.py:15]: A and ahhh
Ok boomer and trying so that it didn't get like overwrite

I shall revert

Comment (HaveFUNRICH) [reply Salad CLI+/options/authentication.py:15]: I shall need
GitHub
101
Guide
By
Mommy Vukky

Comment (Vukkyy) [reply Salad CLI+/options/authentication.py:15]: > and trying so that it didn't get like overwrite
don't worry, i can just fix the conflicts :p

Comment (HaveFUNRICH) [reply Salad CLI+/options/authentication.py]: fixed
e

File: Salad CLI+/Read Me!.txt (hunk 1 above)
Comment (HaveFUNRICH) [reply Salad CLI+/Read Me!.txt:65]: y e s

my brain is just ineperfd
(oh god I can't even write)

Comment (HaveFUNRICH) [reply Salad CLI+/Read Me!.txt:65]: # Done
# e

Comment (HaveFUNRICH) [reply Salad CLI+/Read Me!.txt:65]: I shall go to sleep

Comment (HaveFUNRICH) [reply Salad CLI+/Read Me!.txt:65]: oh wait no I can't I have a giveaway to give
Support Angaros

**Comment by HaveFUNRICH** [comment]
why the fuck did
sIdRefreshToken
and
sRefreshToken
work

**Review by WalkxCode — CHANGES_REQUESTED** [review]
Remove readme.txt changes. #50

**Review by WalkxCode — COMMENTED** [review]
Waiting with merging. See [Discord Messages.](https://canary.discord.com/channels/836251413544828929/836251414304129059/868875197288173618)

**Review by WalkxCode — CHANGES_REQUESTED** [review]
Change `print(f'{Fore.RED}[----------- Profile -----------]')` to `print(f'{Fore.RED}[----------- Profile -----------]\n')`
and `print(f'{Fore.RED}[----------- Session -----------]')` to `print(f'{Fore.RED}[----------- Session -----------]\n')

**Review by WalkxCode — CHANGES_REQUESTED** [review]
Change `'minutes to level up'` to `'minutes left until level up'`

**Comment by HaveFUNRICH** [comment]
ae
check done

**Comment by HaveFUNRICH** [comment]
breh

**Comment by HaveFUNRICH** [comment]
# wtf am I doing

**Comment by WalkxCode** [comment]
> 
> 
> ae
> check done

Is ready to be merged?

**Comment by WalkxCode** [comment]
> 
> # wtf am I doing

Getting brain damage

**Review by WalkxCode — CHANGES_REQUESTED** [review]
Remove `6. Open config.json file in notepad.`???? Not needed???

**Comment by HaveFUNRICH** [comment]
wait waht the fuck when

**Comment by HaveFUNRICH** [comment]
I swear I delete them all

**Comment by HaveFUNRICH** [comment]
oh.. 😳

**Comment by HaveFUNRICH** [comment]
Done ooga booga

**Review by WalkxCode — APPROVED** [review]
woo

**Comment by HaveFUNRICH** [comment]
ae

**Comment by HaveFUNRICH** [comment]
I wonder what revert button do
image

**Pull request merged**